.github/workflows/bundle.yaml

```diff
@@ -0,0 +1,35 @@
+name: bundle
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  bundle:
+    name: bundle
+    runs-on: ubuntu-latest
+    steps:
+    - name: Docker Login
+      if: ${{ github.event_name != 'pull_request' }}
```

Comment:
nit: do we need this `if` check if the workflow is only triggered on new tag pushes defined above?